### PR TITLE
Replace old teal color with new teal color on code.org

### PR DESCRIPTION
**Pull request by kelbyhawn**
Replaces the old teal color `#00ADBC` with the updated teal color `#009EB0` across the https://code.org site.

- Updated all instances with the `var(--brand_primary_default)` variable
- Includes the top main nav bar
- The new teal is slightly darker than the old teal
- Tested by checking each page locally

**Jira ticket:** [ACQ-1440](https://codedotorg.atlassian.net/browse/ACQ-1440)

----

## Before 
Before 

## After
After

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/curriculum/concepts/index.md.erb
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/curriculum/concepts/index.md.erb:3]: This was causing the page to load w/o any styles 🙀

**Review by TurnerRiley — APPROVED** [review]
LGTM! Once you merge please let the DotD know they should expect a lot of eyes changes

**Comment by kelbyhawn** [comment]
> LGTM! Once you merge please let the DotD know they should expect a lot of eyes changes

I have a to-do for that on the Jira ticket 🙌

**Pull request merged**
